### PR TITLE
Shrink menu filebrowser icon restored

### PR DESCRIPTION
**Pull request by masinter**
Fix issue #404 -- restore filebrowser icon

### CODE REVIEW

File: library/FILEBROWSER [hunk 1]
```diff
@@ -179,7 +185,8 @@ You specify how many versions to keep.")))
                                                              (* \; "Setup")
               (FNS FB.STARTUP FB.MAKERIGIDWINDOW)
               (FNS FB.PRINTFN FB.COPYFN))
-        (COMS                                                (* \; "commands and major subfunctions")
+        (COMS                                                (* \; 
+                                                           "commands and major subfunctions")
```
Comment (nbriggs) [inline library/FILEBROWSER:189]: I'd still like to know why this got split onto two lines when, for example, at line 179 above it did not add a line-break.

Comment (masinter) [reply library/FILEBROWSER:189]: i made a separate issue. Once we've fixed the various problems that make the files ugly when printed, we can re-prettyprint everything.

**Review by rmkaplan — APPROVED** [review]
I don't think this should be hung up on the pretty print question.  This does restore the ICONSPEC.  My medley-compare also showed a renaming of a variable (SUB to SI) in one of the subfunctions, but that's immaterial.

**Review by nbriggs — APPROVED** [review]
Yes, this does restore the FB.ICONSPEC.  Other questions can be handled later.

**Pull request merged**